src/WebJobs.Script/Workers/Http/HttpFunctionInvocationDispatcher.cs

```diff
@@ -65,15 +65,14 @@ internal class HttpFunctionInvocationDispatcher : IFunctionInvocationDispatcher
 
         internal Task InitializeHttpWorkerChannelAsync(int attemptCount, CancellationToken cancellationToken = default)
```

Comment:
Thanks for making the change.
I understand the change in `RpcFunctionInvocationDispatcher.cs`. But what is this HttpWorker change for, do we differ the initialization process from other triggers? Or is this change for custom worker?

Reply:
yes! this change will also apply to `BYO workers` via [customHandler](https://docs.microsoft.com/en-us/azure/azure-functions/functions-custom-handlers#:~:text=Custom%20handlers%20are%20lightweight%20web,language%20that's%20not%20officially%20supported.)